charts/trino/values.yaml

```diff
@@ -2,6 +2,13 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# If you want to have multiple Trino Clusters in same namespace 
+# override the following values to avoid naming conflicts
+# Example:
+# coordinatorNameOverride: trino-adhoc 
+# workerNameOverride: trino-adhoc  
+# nameOverride: trino-adhoc
```

Comment:
```suggestion
# -- Override resource names to avoid name conflicts when deploying multiple
# releases in the same namespace.
# @raw
# Example:
# ```yaml
# coordinatorNameOverride: trino-adhoc 
# workerNameOverride: trino-adhoc  
# nameOverride: trino-adhoc
# ```
nameOverride:
coordinatorNameOverride:
workerNameOverride:
```

Reply:
This will also require update to `README.md`, but it should be automatically generated by the `pre-commit` hook, if you've installed it locally.

Reply:
Also fix trailing spaces, see the CI errors for details.